.github/workflows/add-new-impl-versions.yml

```
@@ -0,0 +1,64 @@
+name: Add new implementation versions
+
+on:
+  push:
```

I used push for testing. We should change that. With workflow dispatch, it'll have to be triggered manually. It should work just fine if we put it on a schedule too.

```suggestion
  workflow_dispatch:
```

---

Schedule sounds reasonable. Should be sufficient to run this once a day.

---

```suggestion
  workflow_dispatch:
  schedule:
    # “At minute 36 past every 24th hour.” - https://crontab.guru/#36_*/24_*_*_*
    - cron:  '36 */24 * * *'
```